csrc/executor_kernel_arg.h

```diff
@@ -207,35 +209,36 @@ struct TensorArgAbstract : ArgAbstract {
   std::string toString() const override;
 };
 
-template <typename TENSOR_TYPE, typename nvfuser_index_t>
```

Comment:
Dropped `nvfuser_index_t` from the template parameter list. It's now extracted from `TENSOR_TYPE`